src/include/sof/samples/audio/nxp_detect_test.h

```diff
@@ -0,0 +1,35 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2021 NXP
+ *
+ * Author: Cristina Feies <cristina.ilie@nxp.com>
+ * Author: Viorel Suman <viorel.suman@nxp.com>
+ */
+
+#ifndef __USER_NXP_DETECT_TEST_H__
+#define __USER_NXP_DETECT_TEST_H__
+
+#include <stdint.h>
+#include <sof/lib/wait.h>
+#include <sof/samples/audio/c_models_inc/process.h>
+#include <sof/samples/audio/c_models_inc/preprocess.h>
+#include <sof/samples/audio/c_models_inc/config.h>
+#include <sof/samples/audio/detect_test.h>
+
+#define SILENCE         0
+#define UNKNOWN         1
+#define YES_KEYWORD     2
+#define NO_KEYWORD      3
```

Comment:
Need a prefix

Reply:
Yes, there are some setter /getter functions to access detect_test data necessary in nxp_detect_test.